src/compiler/types.ts

```diff
@@ -7606,6 +7606,8 @@ export interface ResolvedModuleWithFailedLookupLocations {
     affectingLocations?: string[];
     /** @internal */
     resolutionDiagnostics?: Diagnostic[]
+    /** @internal */
+    legacyResult?: string;

```

Comment:
Bikesheddable property name

Reply:
preNodeResult? nodeLegacyResult? legacyNodeResult? node10Result?

I don't like legacyResult at all, **but** I have no idea if any of my suggestions are meaningful.